examples/seismic/self_adjoint/time_blocking.ipynb

```diff
@@ -0,0 +1,1365 @@
+{
```

Comment:
<p><code>gaussing_smooth</code>  should be imported from devito directly</p><p>and same for <code>Derivative</code>  (AFAIR)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1424/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
both of those not used, should have looked more carefully, which I have done now and removed several other unused imports.